api/src/main/java/org/apache/iceberg/expressions/InclusiveMetricsEvaluator.java

```diff
@@ -61,9 +58,7 @@ public InclusiveMetricsEvaluator(Schema schema, Expression unbound) {
   }
 
   public InclusiveMetricsEvaluator(Schema schema, Expression unbound, boolean caseSensitive) {
-    this.schema = schema;
-    this.struct = schema.asStruct();
-    this.caseSensitive = caseSensitive;
+    final StructType struct = schema.asStruct();
```

Comment:
Can you remove the unnecessary `final`?

Reply:
Sure 👍 I agree with this one being unnecessary, but in general, I prefer it to be final by default.